### PR TITLE
Apr 1267 roatp apply api

### PR DESCRIPTION
**Pull request by David-B-Read**

### CODE REVIEW

File: src/SFA.DAS.AdminService.Web/Infrastructure/IRoatpApplicationApiClient.cs [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+    public class RoatpSequence
+    {
+        public int Id { get; set; }
+        public string Title { get; set; }
+        public bool Sequential { get; set; }
+        public List<string> ExcludeSections { get; set; }
+        public List<string> Roles { get; set; }
+    }
+}
```
Comment (martin-hornsby) [inline src/SFA.DAS.AdminService.Web/Infrastructure/IRoatpApplicationApiClient.cs]: These should be in ApplyTypes

File: src/SFA.DAS.AdminService.Web/Views/Roatp/Apply/Applications/_TabularData.cshtml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+﻿@using Newtonsoft.Json
+@model dynamic
+
+@{
+    var tableData = JsonConvert.DeserializeObject<dynamic>(Model);
```
Comment (martin-hornsby) [inline src/SFA.DAS.AdminService.Web/Views/Roatp/Apply/Applications/_TabularData.cshtml]: Can we do anything to move away from dynamic types? I'm presuming not :(

File: src/SFA.DAS.AssessorService.ApplyTypes/ApplicationStatus.cs [hunk 3]
```diff
@@ -9,5 +9,6 @@ public static class ApplicationStatus
         public const string Resubmitted = "Resubmitted";
         public const string Declined = "Declined";
         public const string Approved = "Approved";
+        public const string GatewayAssessed = "GatewayAssessed";
```
Comment (martin-hornsby) [inline src/SFA.DAS.AssessorService.ApplyTypes/ApplicationStatus.cs]: Mentioned elsewhere.. we need to figure out if it goes here. I'm implementing a new Column called GatewayReviewStatus

File: src/SFA.DAS.AdminService.Web/Views/Roatp/Apply/Applications/_TabularData.cshtml (hunk 2 above)
Comment (David-B-Read) [reply src/SFA.DAS.AdminService.Web/Views/Roatp/Apply/Applications/_TabularData.cshtml]: I'd rather not deviate from the view used in QnA/Apply

File: src/SFA.DAS.AdminService.Web/Infrastructure/IRoatpApplicationApiClient.cs (hunk 1 above)
Comment (David-B-Read) [reply src/SFA.DAS.AdminService.Web/Infrastructure/IRoatpApplicationApiClient.cs]: Changes done in APR-1269 branch

**Pull request closed without merging**